OPAL/si/src/main/scala/org/opalj/fpcf/AnalysisScenario.scala

```diff
@@ -33,10 +38,15 @@ class AnalysisScenario[A](val ps: PropertyStore) {
     private[this] var triggeredCS: Set[ComputationSpecification[A]] = Set.empty
     private[this] var transformersCS: Set[ComputationSpecification[A]] = Set.empty
 
+    private[this] var scheduleBatches: List[PhaseConfiguration[A]] = List.empty
+
     private[this] var derivedBy: Map[PropertyKind, (PropertyBounds, Set[ComputationSpecification[A]])] = {
         Map.empty
     }
 
+    private[this] val lazyType = LazyComputation
```

Comment:
Is there a reason for declaring these vals? Looks like the comparisons where they are used could just as easily compare to the values themselves directly.